templates/supervisor.ex.eex

```diff
@@ -0,0 +1,36 @@
+defmodule <%= prj_name %> do
+  use Application
+
+  @connect_timeout 3000
+  def start(_type, _args) do
+    import Supervisor.Spec, warn: false
+
+    children = [
+      worker(<%= prj_name %>.Daemon, [host_port, @connect_timeout]),
+    ]
+
+    opts = [strategy: :one_for_one, name: <%= prj_name %>.Supervisor]
+    Supervisor.start_link(children, opts)
+  end
+
+
+  defp host_port do
+    host = get_env(<%= server_host %>)
+    port = get_env(<%= server_port %>) |> String.to_integer
+    {host, port}
+  end
```

Comment:
Not sure.
All our configurations are in system environment.
It can support {SYSTEM, "ENV_VAR_NAME"} convention.
Don't know if it is worth implementing something that we are never using.